### DISCUSSION TITLE
Search in all fields

### DISCUSSION THREAD

**Discussion by tahayk** [quickwit-oss/quickwit#5172] [General]
I've recently started trying Quickwit, the performance is super awesome, great job.
But now I'm stuck trying to write a search query to perform the following: 
- context: I have created an index with dynamic mapping, this index contains random JSONs, totally unpredictable, and can be nested
- question1: how can I search for a value in all fields without knowing its name in advance ? 
- question2: if I know part of the field's name, how can I search in these fields ?
 
Thank you in advance.

**Comment by fulmicoton** [comment]
There is an unreleased feature (hear "unstable") called "concatenate field"
https://quickwit.io/docs/main-branch/configuration/index-config#concatenate
that can help with question1.

question2 is not possibl today.

**Reply by tahayk** [reply]
Interesting, thank you very much for the help.
